src/SessionMonitor/SessionMonitor.test.js

```diff
@@ -24,4 +24,27 @@ describe('SessionMonitor', () => {
     sessionMonitor.updateSession();
     expect(refreshSessionSpy).toHaveBeenCalledTimes(1);
   });
+
+  it('detects the page correctly', () => {
+    const sessionMonitor = new SessionMonitor(500, 10000000);
+    expect(
+      sessionMonitor.pageFromURL("https://niaid.bionimbus.org/workspace/") 
+    ).toEqual('workspace');
+
+    expect(
+      sessionMonitor.pageFromURL("https://niaid.bionimbus.org/workspace") 
+    ).toEqual('workspace');
+
+    expect(
+      sessionMonitor.pageFromURL("https://niaid.bionimbus.org/dev.html/workspace/") 
+    ).toEqual('workspace');
+
+    expect(
+      sessionMonitor.pageFromURL("data.braincommons.org/login") 
```

Comment:
Thanks! another small one this time: let's not add real sponsor's url here, could you change them to "http://example.com/workspace" etc.?